### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by ankitjain28may**
- [x] Added support to send previous files (backfilling) to STDOUT as well as files
- [x] Provided Formatter as an option to enable
- [x] Code refactoring

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -408,17 +380,27 @@ func (c *CLI) downloadFile(logFile LogFile, ch chan LogFile, customPathOptional
 		params.Marker = aws.String(customPathOptional[0])
 		resp.Marker = aws.String(customPathOptional[0])
 	}
-	// open the out file for writing
-	logrus.Infof("Downloading %s to %s ... ", logFile.LogFileName, logFile.Path)
-	defer logrus.Infof("done\n")
-	if err := os.MkdirAll(path.Dir(logFile.Path), os.ModePerm); err != nil {
-		return logFile, err
+
+	if c.Options.Output == "stdout" {
```
Comment (rajeevrai) [inline cli/cli.go]: change to constants...

File: main.go [hunk 2]
```diff
@@ -26,6 +26,10 @@ import (
 var BuildID string
 
 func main() {
+	appEnv := os.Getenv("APP_ENV")
+	if len(appEnv) > 0 && (appEnv == "dev" || appEnv != "development") {
```
Comment (rajeevrai) [inline main.go]: did not understand ? why is this needed ? also, condition looks wrong. check once

**Review by rajeevrai — CHANGES_REQUESTED** [review]

File: cli/cli.go (hunk 1 above)
Comment (ankitjain28may) [reply cli/cli.go]: okay

File: main.go (hunk 2 above)
Comment (ankitjain28may) [reply main.go]: For local running, I need to pass data from .env file

**Review by rajeevrai — APPROVED** [review]
